autotest/gcore/misc.py

```diff
@@ -314,6 +318,66 @@ def misc_6_internal(datatype, nBands, setDriversDone):
                 dst_ds.GetMetadataItem("", None)
             dst_ds = None
 
+            if (
```

Comment:
I would pull this into a separate test...there is so much going on in "misc_6" already. I would be happy to try a PR if you want.

Reply:
> I would be happy to try a PR if you want.

fine. Yes that's messy